src/libsystemd-network/test-ndisc-rs.c

```diff
@@ -193,6 +193,11 @@ int icmp6_bind_router_solicitation(int index) {
         return test_fd[0];
 }
 
+int icmp6_bind_router_advertisment(int index) {
```

Comment:
The existing code uses the return value to pass the returned socket, so I followed that one. The code handling icmp6_bind_router_solicitation() would then also need a change, but maybe I'll leave that to another patch (and try to get this properly done first).